flutter/lib/button.dart

```diff
@@ -35,6 +36,7 @@ class Button extends StatefulWidget {
       onPressed: onPressed,
       color: kTertiaryColor,
       fontColor: kSecondaryTextColor,
+      buttonHeight: null,
```

Comment:
```dart
factory Button.tertiary({
    Key? key,
    required String text,
    VoidCallback? onPressed,
    double? buttonHeight,
  }) {
    return Button(
      key: key,
      text: text,
      onPressed: onPressed,
      color: kTertiaryColor,
      fontColor: kSecondaryTextColor,
      buttonHeight: buttonHeight,
```

Reply:
fixed according to the suggestion for .secondary and tertiary